### ISSUE TITLE
［系统问题非软件问题］Android 12 Beta 4 v2rayNG 兼容性问题

### ISSUE THREAD

**Issue by lalcd** [2dust/v2rayNG#1180]
### 预期行为
需要代理的 App 能正常使用网络。

### 实际行为
需要代理的 App 无法联网

### 复现方法
1.升级 Android 12 Beta 4
2.打开应用

### 日志信息
08-12 01:24:42.076 D/com.v2ray.v2fly(24937): Process[pid=25025, hasExited=false]
08-12 01:24:42.076 D/com.v2ray.v2fly(24937): /data/user/0/com.v2ray.v2fly/files/sock_path
08-12 01:24:42.083 I/tun2socks(25025): NOTICE(tun2socks): initializing BadVPN tun2socks 1.999.130
08-12 01:24:42.139 D/com.v2ray.v2fly(24937): sendFd tries: 0
08-12 01:24:42.139 I/tun2socks(25025): NOTICE(tun2socks): entering event loop
08-12 01:24:42.238 I/GoLog   (24937): Prepare Result:
08-12 01:24:42.28 I/GoLog   (24937):  Domain:
08-12 01:24:42.238 I/GoLog   (24937):  Port: 
08-12 01:24:42.238 I/GoLog   (24937):  IPs: 
</details>

**Comment by lcyf9102s** [comment]
好像是beta 4的VPN接口有问题，其他类似app都出现一样的情况。

**Comment by d2184** [comment]
Use clash for Android Could  connect to Google, but there will be a little problem when connecting to Twitter and Telegram

**Comment by lalcd** [comment]
确认了，是系统[Bug](https://issuetracker.google.com/issues/196271165)。只有等待修复了。

**Comment by PrimeRay** [comment]
I got the same problem.
My v2rayng works fine on Android 12 before August 12. But my v2rayng can't work after the android 12 Aug.12 update. I have tried all possible settings in v2rayng. 

I had 2 trojan sever. They both works fine on other devices. When I hit run test on v2rayng, one come back with"io: read/write on closed pipe", the other show success and latency 900 ms. But both can't access google or other site.

Device: Pixel 3
System: Android 12 (version 2021 August 12)
User location: China
Method: Trojan
Server location: USA
